contracts/verifier_lsig.teal

```diff
@@ -0,0 +1,47 @@
+#pragma version 8
+
+// Verify not rekeyed
+txn RekeyTo
+global ZeroAddress
+==
+assert
+
+// Determine if the signature is for an address opt-in lsig or open opt-in lsig
+txn GroupIndex
+int 1
++
+gtxns NumAppArgs
+switch _ _ _ open_optin_data addr_optin_data
```

Comment:
I'd recommend giving the `_` label a meaningful name.

Reply:
Renamed to `invalid_arg_count` and move respective `err` branch up in [80cd20e](https://github.com/joe-p/delegated_optin_arc/pull/1/commits/80cd20eaafe9c4f0a649b8b80e511e45e37a9ce2)